storage/cycler.go

```diff
@@ -44,10 +44,12 @@ func (c *Cycler) shiftByKeep(keep int) (first *Package) {
 	return
 }
 
-func (c *Cycler) run(fileKey string, keep int, deletePackage func(fileKey string) error) {
-	c.load()
+func (c *Cycler) run(model string, fileKey string, keep int, deletePackage func(fileKey string) error) {
+	cyclerFileName := path.Join(cyclerPath, model + ".cycler.json")
```

Comment:
```go
path.Join(cyclerPath, "cycler", model + ".json")
```

So, the model cycler filename will be: `.gobackup/cycler/.foo.json` not `.gobackup/foo.cycler.json`